packages/odyssey-react-mui/src/IconWithTooltip.tsx

```diff
@@ -0,0 +1,98 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { memo, ReactElement } from "react";
+import styled from "@emotion/styled";
+import { Tooltip as MuiTooltip } from "@mui/material";
+import type { TooltipProps as MuiTooltipProps } from "@mui/material";
+
+import { HtmlProps } from "./HtmlProps";
+import { InformationCircleIcon } from "./icons.generated";
+import { useMuiProps } from "./MuiPropsContext";
+import {
+  useOdysseyDesignTokens,
+  DesignTokens,
+} from "./OdysseyDesignTokensContext";
+
+const IconContainer = styled("span", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{
+  odysseyDesignTokens: DesignTokens;
+}>`
+  display: inline-flex;
+  border: 1px solid transparent;
+  transition-property: border-color, box-shadow, outline;
+  transition-duration: ${({ odysseyDesignTokens }) =>
+    odysseyDesignTokens.TransitionDurationMain};
+
+  &:focus,
+  &:focus-visible {
+    border-color: ${({ odysseyDesignTokens }) =>
+      odysseyDesignTokens.FocusOutlineColorPrimary};
+    box-shadow: ${({ odysseyDesignTokens }) =>
+      `0 0 0 1px ${odysseyDesignTokens.FocusOutlineColorPrimary}`};
+    outline: ${({ odysseyDesignTokens }) =>
+      `${odysseyDesignTokens.FocusOutlineWidthMain} ${odysseyDesignTokens.FocusOutlineStyle} transparent`};
+    outline-offset: ${({ odysseyDesignTokens }) =>
+      odysseyDesignTokens.FocusOutlineOffsetTight};
+  }
+
+  svg {
+    display: flex;
+  }
+`;
```

Comment:
This, for some reason, causes issues in some consumers of this package. Can you refactor this to be like:
```
const IconContainer = styled("span", {
shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
  })<{
    odysseyDesignTokens: DesignTokens;
  }>(({ odysseyDesignTokens }) => ({
    borderColor: odysseyDesignTokens.FocusOutlineColorPrimary,
    boxShadow: `0 0 0 1px ${odysseyDesignTokens.FocusOutlineColorPrimary}`,
    // ...rest of them
}));
```

Reply:
I can't remember specifically, but they appear when running unit tests with the way it's currently declared